packages/react-composites/tests/browser/chat/fake-adapter/FakeChatAdapterArgs.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { ChatParticipant } from '@azure/communication-chat';
+import { FileMetadata } from '@internal/react-components';
+
+export type FileUpload = FileMetadata & { uploadComplete?: boolean; error?: string; progress?: number };
+
+export type FakeChatAdapterArgs = {
```

Comment:
Let's add docstrings to any non-trivial fields here. For example, looking at this I don't understand what `localParticipantPosition` does. When writing a new test, it would be helpful to be able to understand all these args quickly.